test/e2e/apimachinery/resource_quota.go

```diff
@@ -22,7 +22,7 @@ import (
 	"time"
 
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
**Note for reviewer:** I suspect, recent `gofmt` adds this `v1` but `goimport` doesn't.